### ISSUE TITLE
panic: UNKNOWN TYPE of tftypes.primitive{name:"DynamicPseudoType", _:[]struct {}(nil)}

### ISSUE THREAD

**Issue by dirien** [pulumi/pulumi-terraform-provider#40]
### What happened?

I wanted to "dynamic" bridge the `hashicorp/kubernetes` because I need `kubernetes_annotations` get this panic:

```bash
panic: UNKNOWN TYPE of tftypes.primitive{name:"DynamicPseudoType", _:[]struct {}(nil)}

goroutine 93 [running]:

github.com/pulumi/pulumi-terraform-bridge/pf/proto.element.Type({{0x1028dfc90?, 0x140016813b0?}, 0x90?})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pf/proto/element.go:56 +0x274

github.com/pulumi/pulumi-terraform-bridge/pf/proto.attribute.Type({{{0x14000e862a8, 0x8}, {0x1028dfc90, 0x140016813b0}, 0x0, {0x14000e552c0, 0x51}, 0x1, 0x0, 0x0, ...}})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pf/proto/attribute.go:44 +0x7c

github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/info.MarshalSchemaShim({0x1028ee610, 0x14001508c80})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pkg/tfbridge/info/info.go:831 +0x2c

github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/info.MarshalResourceShim.func1({0x14000e862a8, 0x8}, {0x1028ee610?, 0x14001508c80?})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pkg/tfbridge/info/info.go:868 +0x3c

github.com/pulumi/pulumi-terraform-bridge/pf/proto.blockMap.Range({0x1400124d828?}, 0x1400084c0c0)

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pf/proto/block.go:74 +0xe4

github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/info.MarshalResourceShim({0x1028de6b0, 0x14001e04450})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pkg/tfbridge/info/info.go:867 +0xa4

github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/info.MarshalProviderShim.func2({0x14000e39188, 0x13}, {0x1028de6b0?, 0x14001e04450?})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pkg/tfbridge/info/info.go:936 +0x3c

github.com/pulumi/pulumi-terraform-bridge/pf/proto.resourceMap.Range(0x0?, 0x1400166e8f0)

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pf/proto/resource.go:51 +0x88

github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/info.MarshalProviderShim({0x1028ee838, 0x14000c14450})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pkg/tfbridge/info/info.go:935 +0x10c

github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/info.MarshalProvider(0x1400124daa8)

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pkg/tfbridge/info/info.go:1182 +0x280

github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge.MarshalProviderInfo(...)

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pkg/tfbridge/info.go:267

github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfgen.newInMemoryProvider({_, _}, {_, _, _}, {{0x1028ee838, 0x14000c14450}, {0x14000110749, 0xa}, {0x0, ...}, ...})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pkg/tfgen/pluginHost.go:48 +0x34

github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfgen.NewGenerator({{0x14000110749, 0xa}, {0x14000a24076, 0x6}, {0x101ac174d, 0x6}, {{0x1028ee838, 0x14000c14450}, {0x14000110749, 0xa}, ...}, ...})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pkg/tfgen/generate.go:875 +0x4e8

github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfgen.GenerateSchemaWithOptions({{{0x1028ee838, 0x14000c14450}, {0x14000110749, 0xa}, {0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, ...}, ...})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pkg/tfgen/generate.go:770 +0x11c

main.initialSetup.func2({0x140009bd188?, 0x1028d1238?}, {0x14000b3e4b0?, {0x0?, 0x0?}, 0x0?})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/dynamic/main.go:64 +0xe0

github.com/pulumi/pulumi-terraform-bridge/pf/tfbridge.(*provider).GetSchemaWithContext(0x140009bd188, {0x1028d1238?, 0x14000b3e4b0?}, {0x1003cc02c?, {0x14000c9e120?, 0x91?}, 0x14000b42280?})

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pf/tfbridge/provider.go:227 +0x78

github.com/pulumi/pulumi-terraform-bridge/pf/internal/plugin.(*providerServer).GetSchema(0x14000af3440?, {0x1028d1238?, 0x14000b3e4b0?}, 0x14000b420f0?)

        /home/runner/work/pulumi-terraform-provider/pulumi-terraform-provider/bridge/pf/internal/plugin/provider_server.go:176 +0x1b8

github.com/pulumi/pulumi/sdk/v3/proto/go._ResourceProvider_GetSchema_Handler.func1({0x1028d1238?, 0x14000b3e4b0?}, {0x102704500?, 0x14000b420f0?})

        /home/runner/go/pkg/mod/github.com/pulumi/pulumi/sdk/v3@v3.125.1-0.20240720005025-a3aef6c6522f/proto/go/provider_grpc.pb.go:486 +0xd0

github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc.OpenTracingServerInterceptor.func1({0x1028d1238, 0x14000b3e240}, {0x102704500, 0x14000b420f0}, 0x14000112720, 0x14000948210)

        /home/runner/go/pkg/mod/github.com/grpc-ecosystem/grpc-opentracing@v0.0.0-20180507213350-8e809c8a8645/go/otgrpc/server.go:57 +0x2e4

github.com/pulumi/pulumi/sdk/v3/proto/go._ResourceProvider_GetSchema_Handler({0x102844620, 0x14000af3440}, {0x1028d1238, 0x14000b3e240}, 0x140009b4000, 0x140009e7520)

        /home/runner/go/pkg/mod/github.com/pulumi/pulumi/sdk/v3@v3.125.1-0.20240720005025-a3aef6c6522f/proto/go/provider_grpc.pb.go:488 +0x148

google.golang.org/grpc.(*Server).processUnaryRPC(0x1400095fc00, {0x1028d1238, 0x14000b3e1b0}, {0x1028e49e0, 0x14000cbe000}, 0x14001a78120, 0x14000af34d0, 0x103b7c118, 0x0)

        /home/runner/go/pkg/mod/google.golang.org/grpc@v1.63.2/server.go:1369 +0xb58

google.golang.org/grpc.(*Server).handleStream(0x1400095fc00, {0x1028e49e0, 0x14000cbe000}, 0x14001a78120)

        /home/runner/go/pkg/mod/google.golang.org/grpc@v1.63.2/server.go:1780 +0xb20

google.golang.org/grpc.(*Server).serveStreams.func2.1()

        /home/runner/go/pkg/mod/google.golang.org/grpc@v1.63.2/server.go:1019 +0x8c

created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 84

        /home/runner/go/pkg/mod/google.golang.org/grpc@v1.63.2/server.go:1030 +0x13c

error: failed to get schema: rpc error: code = Unavailable desc = error reading from server: EOF
```

### Example

Try:

```bash
pulumi package add terraform-provider hashicorp/kubernetes
```

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by iwahbe** [comment]
Hi @dirien. This is a duplicate of a prior closed issue: #7, which I believed to be fixed. I'm unable to reproduce on latest (v0.3.0). Can you confirm that you are using terraform-provider v0.3.0:

```console
$ pulumi plugin ls | grep terraform-provider
terraform-provider  resource   0.3.0    78 MB   1 day ago   1 day ago
```

**Comment by iwahbe** [comment]
Closing as no-repo.
